ci/test_full.sh

```diff
@@ -11,3 +11,7 @@ cargo test --verbose
 # test `no_std`
 cargo build --verbose --no-default-features
 cargo test --verbose --no-default-features
+
+# test `i128`
+cargo build --verbose --features=i128
+cargo test --verbose --features=i128
```

Comment:
This will need to be conditioned on `TRAVIS_RUST_VERSION`, so it's only attempted on Rust 1.26+ (currently just beta and nightly).

Reply:
So remove redundant …

```bash
$run cargo build --verbose
```

… from `rustup.sh`?

And add …

```bash
if [[ "$TRAVIS_RUST_VERSION" =~ ^(nightly|beta|stable)$ ]]; then
    cargo build --verbose --features=i128
    cargo test --verbose --features=i128
fi
```

… to `test_full.sh`?

Reply:
`rustup.sh` should be harmless, but you can tweak that if you like.  `test_full.sh` is called directly in `.travis.yml` -- I guess we could remove the redundant build there too.  Doesn't really matter.

The condition looks fine, but I doubt `stable` 1.25 will work yet, as the feature was only recently stabilized for the upcoming 1.26.